components/ecwid/ecwid.app.mjs

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+export default {
+  type: "app",
+  app: "ecwid",
+  propDefinitions: {},
+  methods: {
+    authKeys() {
+      console.log(Object.keys(this.$auth));
+    },
+
+    async getOrders(history = 30, paymentStatus = "PAID", fulfilmentStatus = "AWAITING_PROCESSING") {
+      const fromDate = new Date();
+      fromDate.setDate(fromDate.getDate() - history);
+      const fromDateTimeStamp = Math.floor(fromDate.getTime() / 1000).toString();
+      const res = await axios({
+        url: `https://app.ecwid.com/api/v3/${this.$auth.storeId}/orders?` +
+              `paymentStatus=${paymentStatus}&fulfillmentStatus=${fulfilmentStatus}&createdFrom=${fromDateTimeStamp}`,
+        method: "GET",
+        headers: {
+          "user-agent": "@PipedreamHQ/pipedream v0.1",
+          "accept": "application/json",
+          "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+        },
+      });
+      console.log(res);
+      return res.data.items;
+    },
```

Comment:
Move all this to a method like this: https://github.com/PipedreamHQ/pipedream/blob/c149db25eac78922f9c13e9b249f11576cda513b/components/awork/awork.app.mjs#L14

Reply:
Done.